packages/execution-environments/package.json

```
@@ -0,0 +1,67 @@
+{
+  "name": "@metamask/execution-environments",
```

I think we have an opportunity to improve on our previously established naming conventions here:

```suggestion
  "name": "@metamask/snap-executors",
```